packages/dds/sequence/src/intervalCollection.ts

```diff
@@ -1293,6 +1312,17 @@ export class IntervalCollection<TInterval extends ISerializableInterval>
 		}
 	}
 
+	private getIndex(place: SequencePlace) {
+		if (typeof place === "number") {
+			return place;
+		} else if (place === "start") {
+			return 0;
+		} else if (place === "end") {
+			return this.client?.getLength();
```

Comment:
```suggestion
			return this.client?.getLength() - 1;
```

Perhaps in this case it doesn't matter, but wouldn't you need to convert to a 0-indexed integer?

Reply:
I guess you'd want a saturating subtraction so that the index doesn't go negative in the case of a 0 length string.

Reply:
The index is only used for validation of start <= end. So mapping start to -1 and end to length would be fine.